Description: Fix system lz4 linking with emile
 This needs to go below EFL_LIB_START([Emile]), or else the pkg-config
 get blown away after being setup.
Author: Ross Vandegrift <ross@kallisti.us>
Forwarded: no
Last-Update: 2015-09-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -1107,6 +1107,10 @@
 
 EFL_CHECK_LIBS([EMILE], [zlib])
 
+if test "x${want_liblz4}" = "xyes" ; then
+   EFL_DEPEND_PKG([EMILE], [LIBLZ4], [liblz4])
+fi
+
 EFL_INTERNAL_DEPEND_PKG([EMILE], [eina])
 
 EFL_EVAL_PKGS([EMILE])
